src/toil/provisioners/clusterScaler.py

```diff
@@ -610,30 +610,30 @@ def setNodeCount(self, nodeType, numNodes, preemptable=False, force=False):
                 if delta > 0 and numIgnoredNodes > 0:
                         # We can un-ignore a few nodes to compensate for the additional nodes we want.
                         numNodesToUnignore = min(delta, numIgnoredNodes)
-                        logger.info('Unignoring %i nodes because we want to scale back up again.' % numNodesToUnignore)
+                        logger.debug('Unignoring %i nodes because we want to scale back up again.' % numNodesToUnignore)
                         delta -= numNodesToUnignore
                         for node in ignoredNodes[:numNodesToUnignore]:
                             self.ignoredNodes.remove(node.privateIP)
                             self.leader.batchSystem.unignoreNode(node.privateIP)
                 if delta > 0:
-                    logger.info('Adding %i %s nodes to get to desired cluster size of %i.',
+                    logger.debug('Adding %i %s nodes to get to desired cluster size of %i.',
                                 delta,
                                 'preemptable' if preemptable else 'non-preemptable',
                                 numNodes)
                     numNodes = numCurrentNodes + self._addNodes(nodeType, numNodes=delta,
                                                                 preemptable=preemptable)
                 elif delta < 0:
-                    logger.info('Removing %i %s nodes to get to desired cluster size of %i.', -delta, 'preemptable' if preemptable else 'non-preemptable', numNodes)
+                    logger.debug('Removing %i %s nodes to get to desired cluster size of %i.', -delta, 'preemptable' if preemptable else 'non-preemptable', numNodes)
```

Comment:
I think there is value in leaving this at info (but we should get input from others)

Reply:
I agree.